lib/radius.js

```diff
@@ -449,6 +449,9 @@ Radius.decode_attributes = function(data, attr_hash, vendor, raw_attrs) {
     length = data.readUInt8(1);
     value = data.slice(2, length);
 
+    if (!length)
```

Comment:
Looking at the RFC https://tools.ietf.org/html/rfc2865#section-5 I think this check should be `if (length < 2)`. Since the length includes the "type" and "length" fields themselves, it has to be at least 2 bytes. It doesn't seem like the "value" is allowed to be 0 bytes, but other libraries might accidentally include empty string values as attributes with length zero, so I am fine "supporting" attributes with empty "value".